src/core/grpc_service.proto

```diff
@@ -188,6 +194,62 @@ message HealthResponse
   bool health = 2;
 }
 
+//@@
+//@@.. cpp:var:: message ControlRequest
+//@@
+//@@   Request message for Control gRPC endpoint.
+//@@
+message ControlRequest
+{
+  //@@  .. cpp:enum:: Type
+  //@@
+  //@@     Types of control operation
+  //@@
+  enum Type {
+    //@@    .. cpp:enumerator:: Type::UNLOAD = 0
+    //@@
+    //@@       To unload the specified model.
+    //@@
+    UNLOAD = 0;
+
+    //@@    .. cpp:enumerator:: Type::LOAD = 1 << 0
+    //@@
+    //@@       To load the specified model. Or to reload if the model has
+    //@@       been loaded.
```

Comment:
How about: "To load (or reload) the specified model."

Reply:
it seems to be a bit confusing without elaborating on when "reload" is going to be performed